includes/data-stores/class-wc-admin-reports-data-store.php

```diff
@@ -666,6 +666,24 @@ protected function get_included_products( $query_args ) {
 		return $included_products_str;
 	}
 
+	/**
+	 * Returns comma separated ids of allowed variations, based on query arguments from the user.
+	 *
+	 * @param array $query_args Parameters supplied by the user.
+	 * @return string
+	 */
+	protected function get_included_variations( $query_args ) {
+		$included_variations = array();
+		$operator          = $this->get_match_operator( $query_args );
+
+		if ( isset( $query_args['variations'] ) && is_array( $query_args['variations'] ) && count( $query_args['variations'] ) > 0 ) {
+			$included_variations = $query_args['variations'];
+		}
+
+		$included_variations_str = implode( ',', $included_variations );
```

Comment:
I know the endpoint sanitizes these as integers, but should we be defensive and sanitize here too?

Reply:
I added some extra sanitization here to be safe. https://github.com/woocommerce/wc-admin/pull/1391/commits/275356e15585f88dc9f21d0a676dc6e111633c0d